### PR TITLE
fix-landing-page

### PR DESCRIPTION
**Pull request by theVivek-kumar**
landing page 

image

### CODE REVIEW

File: src/components/Videocard.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+function Card() {
+    return (
+      <>
```
Comment (Debarshi95) [inline src/components/Videocard.js:16]: You can remove the fragment tag, the div is already a container tag

**Review by Debarshi95 — APPROVED** [review]
Good work with the static UI. 
Couldn't find any issue with the code. 
May be start integrating the backend with the UI.

File: src/components/SideMenu.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { FaBars, FaHome, FaHistory } from 'react-icons/fa';
+import { useVideoListing } from '../components/context/videoListingContext'
+import Card from '../components/Videocard'
+import { AiFillLike } from 'react-icons/ai';
+import { MdExplore,MdPlaylistPlay ,MdOutlineWatchLater,MdSlowMotionVideo } from 'react-icons/md';
+import Chip from './chipComponent';
+function SideMenu() {
+    const { videoList } = useVideoListing() 
+    console.log(videoList, "thi id jnjcdjd")
```
Comment (Adarsh-Patel-Dev) [inline src/components/SideMenu.js:10]: remove console before merging

File: src/components/SideMenu.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    // const { video } = videoList
```
Comment (Adarsh-Patel-Dev) [inline src/components/SideMenu.js:11]: remove comments

**Review by Adarsh-Patel-Dev — APPROVED** [review]
UI is looking good and code is well written
You can merge the code after removing comments & console statements

**Pull request merged**